### PR TITLE
Feature/search data testid

### PR DESCRIPTION
**Pull request by nico-olivares**
-Added two data-testid. One to the main search bar and one to the main search button.
-Added three data-path. Facet, date, and number. (per discussion with Alejandro)
-Updated the package.json (version number)
-Updated the package-lock.json (version number)
-Updated the changelog.mdx

### CODE REVIEW

**Comment by decrapifier** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>The README has not been updated. Please update the README.</td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>Could not find any browser results.</td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 08f104935787aab9b22071717bc34039b448714d
</p>

File: src/exampleTypes/TagsQuerySearchBar.js [hunk 1]
```diff
@@ -29,6 +29,7 @@ let buttonStyle = {
 
 let AnimatedButton = ({ disabled, style, className, ...props }) => (
   <Button
+    data-testid="button-main-search"
```
Comment (stellarhoof) [inline src/exampleTypes/TagsQuerySearchBar.js:32]: Instead why not just add the same `data-path` to `SearchBar` and select the input and the search button from there?

**Review by stellarhoof — COMMENTED** [review]
If we're adding `data-path` to some example types we should add it to all the example types components to be consistent. Otherwise we'll have to make follow-up PRs to allow selecting other types of filters.

**Review by warrenv — APPROVED** [review]

**Comment by nico-olivares** [comment]
> If we're adding `data-path` to some example types we should add it to all the example types components to be consistent. Otherwise we'll have to make follow-up PRs to allow selecting other types of filters.

@stellarhoof I agree, but when I did something like that before I got push back for doing too many at once.
Don't you think it would make sense to add a `data-path` to any component that receives `node`?

**Comment by stellarhoof** [comment]
> Don't you think it would make sense to add a `data-path` to any component that receives `node`

Not necessarily. There are components that have access to `node` that are not "example types". Do not worry about the word "example". "Type" refers to the type of a node (`tagsQuery`, `facet`, `number`, etc...). Each type has a component so I don't see adding `data-path` to each example type as overreaching because we know there is simply no good way of selecting filters.

`TagsQuerySearchBar` is not actually a component for a node type but I think it also makes sense to add a `data-path` because there's no good way of selecting it right now.

**Comment by nico-olivares** [comment]
@stellarhoof take a second look. I got rid of the data-testids and added a single data-path to the whole component like you suggested. I also added a couple of other data-paths to what I saw made sense. Let me know if there's any I should get rid of or add.

File: src/exampleTypes/ResultTable/index.js [hunk 2]
```diff
@@ -89,7 +89,7 @@ let ResultTable = ({
   }
 
   return (
-    <>
+    <div data-path={node.path}>
```
Comment (stellarhoof) [inline src/exampleTypes/ResultTable/index.js:92]: Tables are a bit delicate. I don't know if wrapping this one on a div will wreak havoc somewhere. You can pass `data-path` to the `Table` component instead.

Comment (nico-olivares) [reply src/exampleTypes/ResultTable/index.js]: @stellarhoof I moved it to the `Table`

**Review by stellarhoof — APPROVED** [review]

**Comment by nico-olivares** [comment]
@daedalus28 could you please review / merge? I don't know who else has merging permits here. Thanks,

File: src/exampleTypes/TagsQuerySearchBar.js (hunk 1 above)
Comment (nico-olivares) [reply src/exampleTypes/TagsQuerySearchBar.js]: Done.

**Review by dubiousdavid — APPROVED** [review]
Looks good.

File: src/exampleTypes/Date.js [hunk 3]
```diff
@@ -56,7 +56,7 @@ let DateComponent = ({
   )
 
   return (
-    <div>
+    <div data-path={node.path}>
```
Comment (daedalus28) [inline src/exampleTypes/Date.js:59]: These changes might be better generically as part of the contexturify HOCs we have so we don't have to remember to do this on every component

**Pull request merged**